kin/account.py

```diff
@@ -0,0 +1,347 @@
+
+        :param str address: the address of the account to create.
+
+        :param number starting_balance: (optional) the starting XLM balance of the account.
+        If not provided, a default MIN_ACCOUNT_BALANCE will be used.
+
+        # TODO: might want to limit this if we use tx_coloring
+        :param str memo_text: (optional) a text to put into transaction memo, up to 28 chars.
+
+        :return: the hash of the transaction
+        :rtype: str
+
+        :raises: ValueError: if the supplied address has a wrong format.
+        :raises: :class:`KinErrors.AccountExistsError`: if the account already exists.
+        :raises: :class:`KinErrors.MemoTooLongError`: if the memo is longer than 28 characters
+        """
+        tx = self.build_create_account(address, starting_balance=starting_balance, memo_text=memo_text)
+        return self.submit_transaction(tx)
+
+    def send_xlm(self, address, amount, memo_text=None):
+        """Send XLM to the account identified by the provided address.
+
+        :param str address: the account to send XLM to.
+
+        :param number amount: the number of XLM to send.
+
+        # TODO: might want to limit this if we do tx coloring
+        :param str memo_text: (optional) a text to put into transaction memo.
+
+        :return: the hash of the transaction
+        :rtype: str
+
+        :raises: ValueError: if the provided address has a wrong format.
+        :raises: ValueError: if the amount is not positive.
+        :raises: :class:`KinErrors.AccountNotFoundError`: if the account does not exist.
+        :raises: :class:`KinErrors.LowBalanceError`: if there is not enough XLM to send and pay transaction fee.
+        """
+        tx = self._build_send_asset(Asset.native(), address, amount, memo_text)
+        return self.submit_transaction(tx)
+
+    def send_kin(self, address, amount, memo_text=None):
+        """Send KIN to the account identified by the provided address.
+
+        :param str address: the account to send KIN to.
+
+        :param number amount: the amount of KIN to send.
+
+        # TODO: might want to limit this if we do tx coloring
+        :param str memo_text: (optional) a text to put into transaction memo.
+
+        :return: the hash of the transaction
+        :rtype: str
+
+        :raises: ValueError: if the provided address has a wrong format.
+        :raises: ValueError: if the amount is not positive.
+        :raises: :class:`KinErrors.AccountNotFoundError`: if the account does not exist.
+        :raises: :class:`KinErrors.AccountNotActivatedError`: if the account is not activated.
+        :raises: :class:`KinErrors.LowBalanceError`: if there is not enough KIN and XLM to send and pay transaction fee.
+        """
+        tx = self._build_send_asset(self._client.kin_asset, address, amount, memo_text)
+        return self.submit_transaction(tx)
+
+    def build_create_account(self, address, starting_balance=MIN_ACCOUNT_BALANCE, memo_text=None):
```

Comment:
Isn't it an internal method? maybe worth marking it as one (using underscore _)?
same goes for all others internal ones (`build_send_xlm` etc.)

Reply:
Its not, you can see in the readme that these methods have a use of their own, in case you wish to build a transaction but not send it

Reply:
yep, I see it now, all internal are at the end